src/tls.c

```diff
@@ -6266,6 +6266,7 @@ static int TLSX_KeyShare_ProcessDh(WOLFSSL* ssl, KeyShareEntry* keyShareEntry)
     }
 
     /* Derive secret from private key and peer's public key. */
+    ssl->arrays->preMasterSz = ENCRYPT_LEN; /* must provide PMS buffer size */
```

Comment:
Should this size be set at XMALLOC time instead?  If we change the alloc size in the future we won't forget about this one.